tf-inspector/main.tf

```
@@ -0,0 +1,93 @@
+//
+// Module: inspector
+//
+provider "aws" {
+  version = ">= 2.13.0"
+  region  = "${var.region}"
+}
+
+### RESOURCES ###
+# Create Inspector Assessment Target
+resource "aws_inspector_assessment_target" "this" {
+  count = "${var.create_inspector ? 1 : 0}"
+  name  = "${var.name}_assessment_target"
+}
+
+# Create Inspector Assessment Template
+resource "aws_inspector_assessment_template" "this" {
+  count = "${var.create_inspector ? 1 : 0}"
+
+  name       = "${var.name}_assessment_template"
+  target_arn = "${aws_inspector_assessment_target.this.arn}"
+  duration   = "${var.duration}"
+
+  rules_package_arns = [
```

The wrapping brackets should not be necessary, since the data source is already returning a list.

```suggestion
  rules_package_arns = "${data.aws_inspector_rules_packages.this.arns}"
```

---

This actually doesn't work without the wrapping bracket